public/js/src/client.js

```diff
@@ -36,12 +47,13 @@ $(document).ready(function(){
 		var color = cursorColors[cursorId % cursorColors.length];
 		
 		// Adds styles to color how ever many cursors end up being in the document.
-		var css =   '.chunk.listening-'+cursorId+' { position: relative; padding-right: 2px; }\n'+
+// Maybe add some transparency to the cursor:
+		var css =   '.chunk.listening-'+cursorId+' { position: relative; }\n'+
 					'.chunk.listening-'+cursorId+':after { background-color: '+color+'; content: ""; position: absolute; display: inline-block; height: 100%; width: 2px; bottom: 0px; border-radius: 1px; }\n'+
 					'.chat-message-'+cursorId+' { background-color: '+color+'; }\n';
 		
 		// Adds styles if it is the current cursor.		
-		if (cursorId == myCursorId){
+		if (cursorId === myCursorId){
```

Comment:
Whenever possible, use `===` and `!==`.  Using the truthy (`==`) and falsey (`!=`) operators is less performant, and leaves room for error.

All true:
 * `false == 0;`
 * `null == undefined;`
 * `true == 1;`